src/data/get_data.py

```diff
@@ -1,23 +1,23 @@
 from joblib import dump
 
 def main():
-    train = [line.strip() for line in open("../../data/train.txt", "r").readlines()[1:]]
+    train = [line.strip() for line in open("data/train.txt", "r").readlines()[1:]]
     raw_x_train = [line.split("\t")[1] for line in train]
     raw_y_train = [line.split("\t")[0] for line in train]
-    dump(raw_x_train, '../../output/raw_x_train.joblib')
-    dump(raw_y_train, '../../output/raw_y_train.joblib')
+    dump(raw_x_train, 'output/raw_x_train.joblib')
+    dump(raw_y_train, 'output/raw_y_train.joblib')

```

Comment:
This is still dumping things to the local right? Shouldnt it be to the remote smhow